bootstrap/sql/migrations/native/1.3.0/mysql/schemaChanges.sql

```diff
@@ -159,3 +159,8 @@ CREATE TABLE IF NOT EXISTS consumers_dlq (
     timestamp BIGINT UNSIGNED GENERATED ALWAYS AS (json ->> '$.timestamp') NOT NULL,
     UNIQUE(id, extension)
 );
+
+-- Add supportsQueryComment to MSSQL
+update dbservice_entity
+set json = JSON_SET(json, '$.connection.config.supportsQueryComment', true)
+where serviceType = 'Mssql';
```

Comment:
@pmbrull  as supportsQueryComment by default is set to true, if it doesn't find it, it will be set to true, do we need a migration in these cases?

Reply:
Without it the server did not start. I don't think the problem is the value but the server expecting the `key` there I guess